espanol/beatito/beatito.md

```diff
@@ -0,0 +1 @@
+En el rincón donde dormía el beatito fue construyendo un altar, con estampas que le regalaron en la misión y una cruz de xique-xique que el mismo talló y pintó.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.